yescrypt/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "yescrypt"
+version = "0.0.0"
+description = """
+Pure Rust implementation of the yescrypt password hashing function
+"""
+authors = ["RustCrypto Developers"]
+license = "BSD-2-Clause"
```

Comment:
While they're "compatible" in that they're allowed to be used in the same program, that doesn't mean we can arbitrarily relicense code under one license under another.

We could potentially consider changing to `MIT OR Apache-2.0` when the code has been extensively rewritten and is no longer an obvious derived work of the original.

Or we could attempt to get permission from the original authors to relicense it, which is what I've done (with a surprising amount of success) in the past.

Reply:
Yeah... The copyright notices effectively state the same thing, but with slightly different wording. It would be great to have a "greenfield" implementation based on algorithm specification, but I don't know how hard would it be to do compared to the automatic translation (+ cleanup).

Note that right now you state MIT/Apache-2.0 licensing terms in the crate README.